specs/RasterizationScale.md

```diff
@@ -0,0 +1,164 @@
+{
+    CoreWebView2Controller.ShouldDetectMonitorScaleChanges = false;
+    m_webviewAdditionalRasterizationScale = additionalScale;
+    
+    // RasterizationScale is typically the monitor DPI scale and text scaling, but
+    // the app could add an additional scale for its own scenarios.
+    double rasterizationScale = additionalScale * GetDpiScale() * GetTextScale();
+    CoreWebView2Controller.RasterizationScale = rasterizationScale;
+}
+```
+
+
+# API Notes
+See [API Details](#api-details) section below for API reference.
+
+# API Details
+## Win32 C++
+```c#
+interface ICoreWebView2Controller2 : ICoreWebView2Controller {
+  /// The rasterization scale for the WebView. The rasterization scale is the
+  /// combination of the monitor DPI scale and text scaling set by the user.
+  /// This value should be updated when the DPI scale of the app's top level
+  /// window changes (i.e. monitor DPI scale changes or window changes monitor)
+  /// or when the text scale factor of the system changes.
+  ///
+  /// \snippet AppWindow.cpp DPIChanged
+  ///
+  /// \snippet AppWindow.cpp TextScaleChanged1
+  ///
+  /// \snippet AppWindow.cpp TextScaleChanged2
+  ///
+  /// Rasterization scale applies to the WebView content, as well as
+  /// popups, context menus, scroll bars, and so on. Normal app scaling scenarios
+  /// should use the ZoomFactor property or SetBoundsAndZoomFactor API which
+  /// only scale the rendered HTML content and not popups, context menus, scroll bars and so on.
+  ///
+  /// \snippet ViewComponent.cpp RasterizationScale
+  [propget] HRESULT RasterizationScale([out, retval] double* scale);
+  // Set the rasteriation scale property.
+  [propput] HRESULT RasterizationScale([in] double scale);
+
+  /// ShouldDetectMonitorScaleChanges property determines whether the WebView
+  /// attempts to track monitor DPI scale changes. When true, the WebView will
+  /// track monitor DPI scale changes, update the RasterizationScale property,
+  /// and raises RasterizationScaleChanged event. Attempting to set RasterizationScale
+  /// while ShouldDetectMonitorScaleChanges is true will result in RasterizationScaleChanged
+  /// being raised and the value restored to match the monitor DPI scale.
+  /// When false, the WebView will not track monitor DPI scale changes, and the app
+  /// must update the RasterizationScale property itself. RasterizationScaleChanged
+  /// event will never raise when ShouldDetectMonitorScaleChanges is false.
+  [propget] HRESULT ShouldDetectMonitorScaleChanges([out, retval] BOOL* value);
+  /// Set the ShouldDetectMonitorScaleChanges property.
+  [propput] HRESULT ShouldDetectMonitorScaleChanges([in] BOOL value);
+
+  /// Add an event handler for the RasterizationScaleChanged event.
+  /// The event is raised when the WebView detects that the monitor DPI scale
+  /// has changed, ShouldDetectMonitorScaleChanges is true, and the WebView has
+  /// changed the RasterizationScale property.
```

Comment:
From its name, you would think that it is raised whenever the scale changes by whatever means, but from the description, it is raised only when the platform itself initiates the scale change. Can we just raise it whenever the scale changes, for whatever reason?

Reply:
And if the property is renamed (such as to `ActualRasterizationScale`) the event name should change to match.